src/loadCompilerOptions.ts

```diff
@@ -0,0 +1,20 @@
+import * as ts from "typescript";
+
+export const loadCompilerOptions = (): ts.CompilerOptions => {
+  const configFileName = ts.findConfigFile(process.cwd(), ts.sys.fileExists);
+  if (!configFileName) {
+    throw new Error("Could not find config file!");
+  }
```

Comment:
I think that in this case we should just default to disabling features that are dependent on knowing compiler options.